src/swap/components/SwapMessage.tsx

```diff
@@ -16,7 +16,7 @@ export function SwapMessage() {
   }, [error]);
   return (
     <div className="flex">
-      <TextMessage>{message}</TextMessage>
+      <TextLabel2 data-testid="ockSwapMessage_Message">{message}</TextLabel2>
```

Comment:
What is a `TextLabel2` and how is it different from a `TextLabel`? I would rather express variants of similar ideas by name, not arbitrary number.

Reply:
+1 on @kyhyco point

Reply:
@ilikesymmetry, overall what we are optimizing here, it's time to build a component by having clear aligment on how internal Figma primitives are named and than used inside the internal code.

In this case, we want those name to feel more Figma vibes, because they come from the last few years of experience from the internal UI System team in how they approach large complex design systems.

So, something you will see, it's how we expose public components that feel right for our audience, but we build internal components that are more optimized for us as Eng and Design team to work well and fast.

Reply:
To drive it home here, there are meanings associated with new color names (i.e. `foreground`, `forgroundMuted`). And that's what we will see for the properties in figma. The color value is _part_ of the new color names. This helps designers and engineers communicate in a higher level.

I can say "can we change the text color here from foreground to forgroundMuted"? Instead of requesting for specific color values like "red-900" and "blue-500".

Additionally, what are values can we use for text color? Tailwind comes with a lot of colors and we only use a handful. Letting you know what they are is super nice.